files/es/web/javascript/index.md

```diff
@@ -0,0 +1,119 @@
+  - : Explora qué son las APIs y cómo usar algunas de las APIs más comunes que encontrarás a menudo en tu trabajo de desarrollo.
+
+### Guía de JavaScript
+
+- [Guía de JavaScript](/es/docs/Web/JavaScript/Guide)
+  - : Una guía mucho más detallada del lenguaje JavaScript, dirigida a aquellos con experiencia previa en programación, ya sea en JavaScript u otro lenguaje.
+
+### Intermedio
+
+- [Comprendiendo los frameworks JavaScript de lado del cliente](/es/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks)
+  - : Los frameworks de JavaScript son una parte esencial del desarrollo web front-end moderno, ya que proporcionan a los desarrolladores herramientas probadas y comprobadas para crear aplicaciones web interactivas y escalables. Este módulo te brinda algunos conocimientos básicos sobre cómo funcionan los frameworks del lado del cliente y cómo encajan en tu conjunto de herramientas, antes de pasar a la serie de tutoriales que cubren algunos de los más populares en la actualidad.
+- [Una reintroducción a JavaScript](/es/docs/Web/JavaScript/A_re-introduction_to_JavaScript)
+  - : Una descripción general para aquellos que _piensan_ que saben JavaScript.
+
+- [Estructuras de datos en JavaScript](/es/docs/Web/JavaScript/Data_structures)
+  - : Descripción general de las estructuras de datos disponibles en JavaScript.
+- [Comparaciones de igualdad y similaridad](/es/docs/Web/JavaScript/Equality_comparisons_and_sameness)
+  - : JavaScript proporciona tres diferentes operadores de comparación de valores: igualdad estricta usando `===`, igualdad abstracta usando `==`, y el método {{jsxref("Global_Objects/Object/is", "Object.is()")}}.
+- [Cierres (clousures)](/es/docs/Web/JavaScript/Closures)
+  - : Un cierre es la combinación de una función y un entorno léxico dentro del cual se declaró esa función.
+
+### Avanzado
+
+- [Herencia y la cadena de prototipos](/es/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)
+  - : Explicación de la herencia basada en prototipos ampliamente incomprendida y subestimada.
+- [Modo estricto](/es/docs/Web/JavaScript/Reference/Strict_mode)
+  - : El modo estricto define que no puedes usar ninguna variable antes de iniciarla. Es una variable restringida de ECMAScript 5, para un rendimiento más rápido y una depuración más fácil.
+- [Arreglos escritos en JavaScript](/es/docs/Web/JavaScript/Typed_arrays)
+  - : Los arreglos escritos en JavaScript proveen un mecanismo para acceder a datos binarios sin formato.
+- [Gestión de la Memoria](/es/docs/Web/JavaScript/Memory_Management)
+  - : Ciclo de vida de la memoria y recolección de basura en JavaScript.
+- [Modelo de Concurrencia y Bucle de Eventos](/es/docs/Web/JavaScript/EventLoop)
+  - : JavaScript tiene un modelo de concurrencia basado en un "bucle de eventos".
+
+## Referencia
+
+Consulta la documentación completa de la [referencia de JavaScript](/es/docs/Web/JavaScript/Reference).
+
+
```

Comment:
Being a little picky, but there's an extra newline here

Reply:
I have seen it but I decided not to say anything since now we are two, I think it's time to do it.

Do you know a good `Prettier`, `linter`, or `auto-formatted for markdown? which editor?

Reply:
There are lots out there, I've recently been trying out [`markdownlint`](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) in VS Code